### PR TITLE
Fix app run crash

### PR DESCRIPTION
**Pull request by M-Adoo**

### CODE REVIEW

File: core/src/builtin_widgets/theme/custom_styles.rs [hunk 1]
```diff
@@ -24,20 +22,24 @@ macro_rules! fill_custom_style {
...
-      .unwrap_or_else(|| {
-        panic!(
-          "The custom theme({}) is not init in theme, use it after init.",
-          std::any::type_name::<Self>()
-        )
-      })
+
+    c.unwrap_or_else(|| {
+      let style = Self::default_style(ctx);
+      let Theme::Full(app_theme) = ctx.app_theme_mut() else { unreachable!() };
```
Comment (sologeek) [inline core/src/builtin_widgets/theme/custom_styles.rs:37]: Why need `else` here?

Comment (M-Adoo) [reply core/src/builtin_widgets/theme/custom_styles.rs:37]: Only a `let else` syntax here to only accept a full theme, then I can omit an if else branch.

File: widgets/src/themes.rs [hunk 2]
```diff
@@ -1,34 +1,2 @@
 mod styles_sheet;
-use ribir_core::{fill_svgs, prelude::*};
 pub use styles_sheet::*;
-
-pub(crate) fn add_to_theme(theme: &mut FullTheme) {
-  fill_svgs! {
-    theme.icon_theme,
-    svgs::ADD: "./themes/icons/add_FILL0_wght400_GRAD0_opsz48.svg",
```
Comment (sologeek) [inline widgets/src/themes.rs:8]: remove here, if developer don't use material theme, it only use default theme, it will not found svg file.

**Review by wjian23 — APPROVED** [review]

File: widgets/src/themes.rs (hunk 2 above)
Comment (M-Adoo) [reply widgets/src/themes.rs:8]: I'll move these icons into core.

**Review by sologeek — APPROVED** [review]

**Pull request merged**